sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -175,8 +177,9 @@ def _add_argparse_args(cls, parser):
        'Or to take arms against a sea of troubles, '])
    | beam.io.WriteToText(my_options.output))
 
-  p.run()
+  result = p.run()
```

Comment:
Looks good. I wonder if these should be part of the snippets (including constructing an empty PipelineOptions()) but we should address that as part of the documentation revamp.